packages/sources/universal/src/endpoint/sandbox.ts

```diff
@@ -0,0 +1,105 @@
+import {
+  AdapterResponse,
+  Method,
+  Requester,
+  Validator,
+  AdapterError,
+} from '@chainlink/ea-bootstrap'
+import { Config, ExecuteWithConfig, InputParameters } from '@chainlink/ea-bootstrap'
+
+export const supportedEndpoints = ['sandbox']
+
+export interface ResponseSchema {
+  success?: hexstring
+  error?: string
+}
+
+export const description = 'This is an example endpoint description for example adapter.'
+
+export type TInputParameters = {
+  code: string
+  httpQueries?: HttpQuery[]
+  args?: string[]
+  secrets?: string
+}
+
+export const inputParameters: InputParameters<TInputParameters> = {
+  code: {
+    description: 'JavaScript source code to be executed',
+    required: true,
+  },
+  httpQueries: {
+    description: 'HTTP queries to be performed and passed to the source code',
+  },
+  secrets: {
+    description: 'Bytes representing an encrypted secrets object',
+  },
+  args: {
+    description: 'Array of on-chain arguments which are passed to the source code',
+  },
+}
+
+type HttpQuery = {
+  HttpVerb: 'Get'
+  url: string
+  headers: Record<string, string>
+}
+
+// a hexstring will start with '0x'
+type hexstring = string
+
+export const execute: ExecuteWithConfig<Config> = async (request /*, _, config*/) => {
+  const validator = new Validator(request, inputParameters)
+
+  const jobRunID = validator.validated.id
+
+  const httpQueries = validator.validated.data.httpQueries
+  const code = validator.validated.data.code
+  const secrets = validator.validated.data.secrets
+  const args = validator.validated.data.args
+
+  const data = { httpQueries, code, secrets, args }
+
+  // How do I create an new adapter-specfic config var instead of accessing process.env?
+  const url = process.env['SANDBOX_URL']
+
+  if (!url)
+    throw new AdapterError({
+      jobRunID: request.id,
+      message: `set the URL of the FaaS sandbox using the SANDBOX_URL config variable`,
+      statusCode: 500,
+    })
+
+  // TODO: inline-replace any instance of the string `$(secrets[x])` within the URL & headers w/ decrypted secrets
+
+  const options = { url, method: 'POST' as Method, data }
+
+  const sandboxResponse = await Requester.request<ResponseSchema>(options)
+
+  const adapterResponse = {
```

Comment:
Should follow existing patterns for handling responses (`Requester.success` and `Requester.withResult`). Might make this next bit of code cleaner.

Reply:
Requester.withResult did not support returning an object containing a single `success` or `error` field, hence why I did not use it.  If you would prefer, I can make the required modification to the core framework to achieve this functionality.

Reply:
Resolved comment as discussed in DMs.  Requester.withResult does not support returning an object.